src/styles/blocks/card.scss

```diff
@@ -0,0 +1,54 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
+  height: 407px;
+  border: 1px solid $grey-star-color;
```

Comment:
Ensure that the color value for the border is defined as a variable in a separate file, as per the task requirements for maintainability.